packages/model/src/calendar.ts

```diff
@@ -6,6 +6,7 @@ import { WorkingGroupDataObject, WorkingGroupResponse } from './working-group';
 export interface CalendarDataObject extends BasicCalendar {
   groups?: Pick<GroupDataObject, 'id' | 'active'>[];
   workingGroups?: Pick<WorkingGroupDataObject, 'id' | 'complete'>[];
+  associatedGoogleCalendarId?: string;
```

Comment:
any point of exposing this outside of the data provider? Especially since I don't think Squidex has it?

Reply:
it was not necessary to expose this field, thanks :)
committed here https://github.com/yldio/asap-hub/pull/3046/commits/054e472ba727557a5110c9da091b34de9f3b5a37